dev-docs/source/Testing/MuonInterface/MuonTesting.rst

```diff
@@ -1,224 +1,190 @@
+:ref:`manage user directories <ManageUserDirectories>`.
 
-   -  Beside ``Runs`` enter ``51341-3`` and check ``Simultaneous`` below
-   -  Right click the ``Fit Function`` table area; select
-      ``Add Function``
-   -  Add ``Background`` > ``Flat Background``
-   -  Add ``Muon`` > ``DynamicKuboToyabe``
-   -  Check the ``Global`` box for: ``A0``, ``Asym``, ``Delta``, ``Nu``
-   -  Click on the value ``0.0000`` for ``Field`` - a ``...`` box should
-      appear, click it
+-------------------------
 
-      -  On the ``Set`` dropdown, select ``Fix All``
-      -  Enter values of ``0``, ``5``, ``10`` for the three runs
+.. _ionic_diffusion_test:
 
-   -  Set ``A0 = 0.05``, ``Asym = 0.15``, ``Delta = 0.2``, ``Nu = 0.1``
-   -  Click ``Fit`` > ``Fit``
-
--  In the ``Results Table`` tab
-
-   -  Under ``Fitting Results`` check the ``Simultaneous`` button
-   -  Click ``Create Table``
-   -  A table should appear in the main Mantid work area
-
--  In the ``Results Table`` workspace
-
-   -  Expected values (similar to within 50%) ..
-   -  f0.A0: -0.083439833711330152
-   -  f1.Asym: 0.19125607982528858
-   -  f1:Delta: 0.3301445010124604
-   -  f1:Nu: 0.8574644977974730220884
-
--  In the workspaces section of the main Mantid window:
-
-   -  Click of the ``MuonSimulFit_5141-3`` group
-   -  Within this there should be three workspaces
-   -  Right-click each workspace
-   -  Choose ``Plot spectrum``
-   -  Choose ``1D plot`` and click ``Plot All``
+Ionic Diffusion Test
+--------------------
 
-Possible problems
-^^^^^^^^^^^^^^^^^
+**Time required 5 - 10 minutes**
 
--  Loading local data for 51341-3; these may not be in recognised
-   directories - make sure that this is the case.
+- Open **Muon Analysis** (*Interfaces* > *Muon* > *Muon Analysis*)
+- Change *Instrument* to **EMU**, found in the *Home* tab
+- Load runs ``51341-3``
+- Go to the **Fitting** tab
+	- Check the **Simultaneous fit over** checkbox, and change from **Run** 
+	  to **Group/Pair**
+	- Right click the empty table area; Select **Add Function**
+	- Add a **Flat Background** (*Background* > *Flat Background*)
+	- Similarly, add **DynamicKuboToyabe** (*Muon* > *MuonGeneric* > 
+	  *DynamicKuboToyabe*)
+	- Check the **Global** checkbox for the parameters **A0**, **Asym**, 
+	  **Delta** and **Nu**
+	- Set ``A0 = 0.05``, ``Asym = 0.15``, ``Delta = 0.2`` and ``Nu = 0.1``
+	- Click the value for the **Field** parameter; A ``...`` should appear next 
+	  to it, click it. A new window should appear
+	- On the **Set** dropdown menu, select **Fix All**
+	- Enter values ``0``, ``5`` and ``10`` for each run respectively
+	- Click Fit
+- Go to the **Results** tab
+	- At the bottom of the interface, click **Output Results**
+	- A table should appear in the workspaces toolbox called *Results Table*
+- Click on the main Workbench window and open the results table
+- Expected Values are (similar to within 50%):
+	- **f0.A0:** ``-0.083439833711330152``
```

Comment:
just do 3 significant figures and update the values. Nu seems the most different (but the fits are good)